index.html

```diff
@@ -1418,6 +1461,9 @@
           </ul>
         </li>
         <li>
+          Let <var>options</var> be the second argument.
```

Comment:
The more verbose style is consistent with e.g. the HTML spec. I suggest we prefer clarity over brevity. But I don't feel strongly, so leaving it to the editors.